examples/tuning/cta_svm/results/pipeline/best_test_acc.csv

```diff
@@ -0,0 +1,4 @@
+id,acc,_step,_wandb_runtime,_runtime,test_acc,_timestamp,pipeline.0,pipeline.1,pipeline.2
```

Comment:
Among all pipelines (missing allowed), the three configurations with the highest test_acc